config/subresource-integrity.php

```diff
@@ -1,13 +1,54 @@
 <?php
 
 return [
+    /*
+    |--------------------------------------------------------------------------
+    | Assets Repository Path
+    |--------------------------------------------------------------------------
+    |
+    | This path contains all the asset files used by your application.
+    |
+    */
+
     'base_path' => base_path('public'),
 
+    /*
+    |--------------------------------------------------------------------------
+    | Default Hash Algorithm
+    |--------------------------------------------------------------------------
+    |
+    | This option controls the default algorithm that will be used to hash
+    | assets files used by your application. By default, the sha256 algorithm is
+    | used; however, you remain free to modify this option if you wish.
+    |
+    | Supported: "sha256", "sha384", "sha512"
+    |
+    */
+
     'algorithm' => env('SRI_ALGORITHM', 'sha256'),
 
+    /*
+    |--------------------------------------------------------------------------
+    | Mix SRI Path
+    |--------------------------------------------------------------------------
+    |
+    | This path contains 'mix-sri.json' file generated by Laravel Mix extension
+    | 'laravel-mix-sri' witch contains generated hashes on build.
```

Comment:
I wouldn't explicitly name my `laravel-mix` extension in the comments as the only possibility.

A better option would be to say that it can be generated by webpack (and laravel-mix) using extensions like `laravel-mix-sri`.